src/Surfnet/StepupMiddleware/ApiBundle/Tests/Configuration/Entity/InstitutionConfigurationOptionsTest.php

```diff
@@ -0,0 +1,89 @@
+<?php
+
+/**
+ * Copyright 2016 SURFnet B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Surfnet\StepupMiddleware\ApiBundle\Tests\Configuration\Entity;
+
+use PHPUnit_Framework_TestCase as TestCase;
+use Surfnet\Stepup\Configuration\Value\Institution;
+use Surfnet\Stepup\Configuration\Value\ShowRaaContactInformationOption;
+use Surfnet\Stepup\Configuration\Value\UseRaLocationsOption;
+use Surfnet\StepupMiddleware\ApiBundle\Configuration\Entity\InstitutionConfigurationOptions;
+
+class InstitutionConfigurationOptionsTest extends TestCase
+{
+    /**
+     * @test
+     * @group entity
+     */
+    public function serialized_institution_configuration_options_have_the_correct_keys()
```

Comment:
same as above, it seems to me this could be a single test, verifying a single behaviour (also, think about the test names)